misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -2307,19 +2352,10 @@ public AnnotationDeclarationNode transform(AnnotationDeclarationNode annotationD
     @Override
     public AnnotationAttachPointNode transform(AnnotationAttachPointNode annotationAttachPointNode) {
         Token sourceKeyword = formatToken(annotationAttachPointNode.sourceKeyword().orElse(null), 1, 0);
-//        Token firstIdent;
-//        if (annotationAttachPointNode.secondIdent().isPresent()) {
-//            firstIdent = formatToken(annotationAttachPointNode.firstIdent(), 1, 0);
-//        } else {
-//            firstIdent = formatToken(annotationAttachPointNode.firstIdent(), env.trailingWS, env.trailingNL);
-//        }
-//
-//        Token secondIdent = formatToken(annotationAttachPointNode.secondIdent().orElse(null), env.trailingWS,
-//                env.trailingNL);
+        NodeList<Token> identifiers = formatNodeList(annotationAttachPointNode.identifiers(), 1, 0, 0, 0);
```

Comment:
Can we change this to the following?
```suggestion
        NodeList<Token> identifiers = formatNodeList(annotationAttachPointNode.identifiers(), 1, 0, env.trailingWS, env.trailingNL);
```

Reply:
Fixed.